packages/wonder-blocks-modal/components/focus-trap.js

```diff
@@ -18,48 +20,63 @@ import * as ReactDOM from "react-dom";
  *     actually coupled to the modal, and these could be renamed "children"
  *     instead if we were to generalize!
  */
-export default class FocusTrap extends React.Component<{children: React.Node}> {
+
+type Props = {|
+    children: React.Node,
+|};
+
+export default class FocusTrap extends React.Component<Props> {
     /** The most recent node _inside this component_ to receive focus. */
-    _lastNodeFocusedInModal: ?Node;
+    lastNodeFocusedInModal: ?Node;
 
     /**
      * Whether we're currently applying programmatic focus, and should therefore
      * ignore focus change events.
      */
-    _ignoreFocusChanges: boolean;
+    ignoreFocusChanges: boolean;
+
+    modalRoot: Node;
```

Comment:
Can you add a comment for `modalRoot` explaining what it's used for?

Reply:
I'm also surprised that Flow is ok with this. Why do we need a `?Node` for `lastNodeFocusedInModal` but not for `modalRoot`? It does seem like there is the potential for it to be undefined, correct?

Reply:
We should probably also do `this.modalRoot = null;` in the constructor, too.

Reply:
I can change it to ?Node... I wonder why `flow` doesn't complain about this.